packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -84,21 +84,7 @@ export class LayeredRuntimeResolverCreator {
         resolvers.Query[findField] = (parent: any, args: any, context: GraphbackContext) => {
           return context.graphback[modelName].findBy(args.filter, args.orderBy, args.page, context)
         }
-      }
-
-      // If delta marker is encountered, add resolver for `delta` query
-      if (resolverElement.config.deltaSync) {
-        const deltaQuery = getDeltaQuery(resolverElement.graphqlType.name)
-
-        resolvers.Query[deltaQuery] = async (parent: any, args: any, context: GraphbackContext) => {
-          const dataSyncService: any = context.graphback[modelName];
-
-          if (dataSyncService.sync === undefined) {
```

Comment:
This would technically work with some other class that happens to have the `.sync` property or function. You should use `instanceof` to ensure the class and give full typing

```ts
function isDataSyncService(service: any): service is DataSyncCRUDService {
  return service instanceof DataSyncCRUDService
}
```

Reply:
Fair point, Thanks!